AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb

```diff
@@ -612,6 +656,7 @@
     "modin_time = time.time() - t1\n",
     "\n",
     "print(\"Pandas Time(seconds):\",pandas_time,\"\\nModin Time(seconds):\",modin_time)\n",
+    "verify_and_print_times(pandas_time, modin_time)\n",
```

Comment:
I don't see a return result from this function for read_csv.

Reply:
Where do you expect to see the return result? `verify_and_print_times` returns void, the function's only effect is that it prints time results on the screen

Reply:
I don't see any print from this function in the notebook (when opening `...` -> `View file`).

Reply:
Oh, that's because I haven't saved the new output of the notebook's cells. Rerunning the notebook with saving the new output would probably change execution times that were printed before, as I would probably be running the notebook on a different version of Modin and on a different machine. @praveenkk123, can I just rerun the notebook on an arbitrary machine and update the execution times? Wouldn't this cause any legal problems?